content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -188,6 +188,43 @@ The following plugins support `WaitForFirstConsumer` with pre-created Persistent
 and pre-created PVs, but you'll need to look at the documentation for a specific CSI driver
 to see its supported topology keys and examples.
 
+:::note
+   If you choose to use `waitForFirstConsumer` in storage classes, do not use `nodeName` in the Pod spec
+   to specify node affinity. If `nodeName` is used in this case, then PVC will remain in `pending` state. For more details refer https://github.com/openebs/openebs/issues/2915.
+
+   Instead, you can use node selector for hostname in this case as shown below
+:::
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: task-pv-pod
+spec:
+  affinity:
```

Comment:
It could be even simpler than this if you use `nodeSelector`

Reply:
updated